commons-test/src/main/java/org/infinispan/commons/test/TestSuiteProgress.java

```diff
@@ -13,66 +13,79 @@
  * @since 9.0
  */
 public class TestSuiteProgress {
-
    private static final Logger log = Logger.getLogger(TestSuiteProgress.class);
+   private static final char ESC = (char) 27;
+   private static final String RED = string(ESC, '[', '3', '1', 'm');
```

Comment:
Anything wrong with ```RED = "\u001b[31m"``` ?

Reply:
Actually the way I'm doing it is a leftover from some preprocessing I had to play nice with Surefire's way of encoding special chars. I'll fix that.